avocado/utils/process.py

```diff
@@ -1189,4 +1194,6 @@ def system_output(cmd, timeout=None, verbose=True, ignore_status=False,
     cmd_result = run(cmd=cmd, timeout=timeout, verbose=verbose, ignore_status=ignore_status,
                      allow_output_check=allow_output_check, shell=shell, env=env,
                      sudo=sudo, ignore_bg_processes=ignore_bg_processes)
+    if strip_trail_nl:
```

Comment:
This code is consistent with the commit description.  The original card, though, has the intention to actually change the current behavior and align it with the other similar utility functions given.

I suggest you change this parameter to something like `dont_strip_nl` with the opposite behavior.  IMO, I'd even consider such a parameter to be a candidate for removal in a future release, say, before the next LTS release.